### ISSUE TITLE
Compatibilty with version 2.9.2 and old seeds

### ISSUE THREAD

**Issue by AntonioLaTrippa** [fyookball/electrum#394]
Hi Fyookball,
My platform is Windows 10. I have tried transferring some BCH from the wallet I created after the fork using v 2.9.2, but v 2.9.2 can't connect to the servers anymore. I then downloaded v 3.0, and created a new wallet with the same seed I sued to create the wallet in v 2.9.2. This new wallet is empty, meaning that not only it doesn't contain any BCH, but also has no operations or transactions in it. Also, if I try to open the old wallet with v 3.0, it crashes without giving any explanation about the reason. It just says that Windows (I use 10) is sending some data. I then downloaded v 2.9.4 and did exactly the same, with exactly the same results. Actually, v 2.9.4 doesn't even do anything after it crashes, it just crashes and that's it.
How do I retrieve now my BCH?
Thank you for your help.

**Comment by AntonioLaTrippa** [comment]
P.S.: V 2.9.2 perfectly opens the wallet and all the BCH and the previous operations done are there, it just doesn't do anything because it cannot connect.

**Comment by AntonioLaTrippa** [comment]
Something more. I created a new "watching only" wallet using the addresses that the v 2.9.2 created, extracting them from there. All my BCH are there. So I tried and created a new functioning wallet with the keys I extracted from the old wallet, but the new wallet is empty. If I could add the old addresses to this new wallet I believe my issues would be sorted. Problem is I am not able to do it. I tried doing this:

http://electroncash.readthedocs.io/en/latest/old/faq.html#how-can-i-pre-generate-new-addresses

But it doesn't work. Probably I don't know the correct syntax.
